.github/workflows/release.yml

```diff
@@ -13,51 +7,60 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'release version in format 7.y.z'
+        description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
+        default: '7.y.z'
   push:
     branches: 
       - 'release'
     paths:
       - 'VERSION'
 
 jobs:
-  release-build-publish:
+  build:
+    name: Create Che Devfile Registry Release
     runs-on: ubuntu-latest
     steps:
-      - name: Clone source code
-        uses: actions/checkout@v1
+      -
+        name: "Checkout  source code"
+        uses: actions/checkout@v2
```

Comment:
Any particular reason to use `v2` over `v1`?

Reply:
@ericwill no as such, Please let me know do you require v1 over v2?

Reply:
I don't think so, I see we are using `fetch-depth` so it shouldn't make a difference.